### PR TITLE
Update README.md

### LINKED ISSUE #505: [Docs]: Improving README.md
### what's wrong in the documentation?

We need to add : 
+ restructure the heading to make it less crammy
+ formatting text for better look
+ styling text for better impact

Hey @priyankarpal, I have already worked on the README before and I think I can improve it further !
Do assign me and add the GSSOC label as I am not aware of the working yet 😅

### Add screenshots

Add screenshots

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by the-amazing-atharva**
## Related Issue

fixes #505 
for GSSOC'23
## Suggestion

@priyankarpal I think we should add the Ph. logo here can you provide that in this comment section?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,21 @@
...
 
-![projectshut](https://user-images.githubusercontent.com/88102392/235369195-189ad8cd-31df-4099-8b99-3efac3056651.png)
 
-## 🤓 Why ProjectsHut ?
+![projectshut](https://user-images.githubusercontent.com/88102392/235369195-189ad8cd-31df-4099-8b99-3efac3056651.png)
 
-Welcome to ProjectsHut, a cutting-edge web platform for publishing projects without any cost. Our platform is designed to provide individuals with an exceptional space to showcase their work and engage with a worldwide audience.
+## 🚀 About ProjectsHut
++ Welcome to **[ProjectsHut](https://projectshut.vercel.app)**, a *cutting-edge web platform* for publishing projects without any cost. 
++ Our platform is designed to provide individuals with an exceptional space to showcase their work and engage with a worldwide audience.
```
Comment (priyankarpal) [inline README.md:10]: remove these lines

File: README.md [hunk 2]
```diff
@@ -28,14 +31,15 @@ Contributing to open-source software (OSS) projects can be a rewarding and fulfi
...
-
-Furthermore, utilizing React and Tailwind CSS provides many benefits, including easy maintenance and scalability of the application. This ensures that our platform can adapt and grow alongside the needs of our users, while maintaining its superior performance and polished aesthetics.
++ We're excited to share that our project leverages the power of React for the frontend, and the stylish and versatile **Tailwind CSS** for an eye-catching and intuitive user interface. 
++ The seamless integration of these two cutting-edge technologies has enabled us to create a responsive and visually stunning platform that is both easy to use and navigate.
++ Furthermore, utilizing React and Tailwind CSS provides many benefits, including easy maintenance and scalability of the application. 
++ This ensures that our platform can adapt and grow alongside the needs of our users, while maintaining its superior performance and polished aesthetics.
```
Comment (priyankarpal) [inline README.md:42]: also remove these lines 
```suggestion

```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Comment by the-amazing-atharva** [comment]
Done @priyankarpal

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

**Pull request merged**